ipn/ipnstate/ipnstate.go

```diff
@@ -51,8 +47,11 @@ type Status struct {
 	// There are no surrounding dots.
 	// MagicDNSSuffix should be populated regardless of whether a domain
 	// has MagicDNS enabled.
+	// This field is duplicated into Tailnet.MagicDNSSuffix.
 	MagicDNSSuffix string
 
+	Tailnet TailnetStatus
```

Comment:
Make it a pointer so it's JSON null when you're logged out? But then document what nil here means.
 
Otherwise you need to document the meaning of Tailnet.Name == "".